cou/apps/auxiliary.py

```diff
@@ -566,7 +566,7 @@ async def _wait_for_sealed_status(self) -> None:
             raise_on_error=False,
         )
 
-        app_status = await self.model.get_application_status(app_name=self.name)
+        app_status = self.status
```

Comment:
We need to get current status and like this will not have the old status?

Reply:
Thanks! Maybe I will refactoring this part later

Reply:
reverted